HoLC/Handlers/FileHandler.cpp

```diff
@@ -442,6 +442,16 @@ bool FileHandler::hasFile(QString filePath)
     return false;
 }
 
+void FileHandler::reloadFile()
+{
+    QString path = mpCurrentFile->mFileInfo.absoluteFilePath();
+    FileObject::FileTypeEnum type = mpCurrentFile->mType;
+    delete mpCurrentFile;
+    mFilePtrs.removeAll(mpCurrentFile);
```

Comment:
This is the place that fails on travis. It is using Qt4 and removeAll is apparently not available for QVector yet.